front/app/components/PostShowComponents/Comments/utils.ts

```diff
@@ -0,0 +1,13 @@
+export function getCommentContent(commentBody: string) {
+  return commentBody.replace(
+    /<span\sclass="cl-mention-user"[\S\s]*?data-user-id="([\S\s]*?)"[\S\s]*?data-user-slug="([\S\s]*?)"[\S\s]*?>([\S\s]*?)<\/span>/gi,
+    '<a class="mention" data-link="/profile/$2" href="/profile/$2">$3</a>'
+  );
+}
+
+export function getEditableCommentContent(commentBody: string) {
+  return commentBody.replace(
+    /<span\sclass="cl-mention-user"[\S\s]*?data-user-id="([\S\s]*?)"[\S\s]*?data-user-slug="([\S\s]*?)"[\S\s]*?>@([\S\s]*?)<\/span>/gi,
```

Comment:
So, the only difference between editable and not editable comments is this `@` sign, isn't it? Why don't we include it in both cases?

Reply:
Not sure, I don't have the full context on the decision. I just cleaned it up a bit and left the old implementation. Leaving out of scope for this PR